lib/core/migrate/migrate.js

```diff
@@ -372,6 +372,7 @@ const migrations = [
 			await Migrators.moveOption('showImages', 'display memedad', 'showImages', 'display_memedad');
 			await Migrators.moveOption('showImages', 'display ridewithgps', 'showImages', 'display_ridewithgps');
 			await Migrators.moveOption('showImages', 'display photobucket', 'showImages', 'display_photobucket');
+			await Migrators.moveOption('showImages', 'display peertube', 'showImages', 'display_peertube');
```

Comment:
@BenMcGarry, yeah, the option `display_peertube` is automatically generated, so this isn't necessary.

Reply:
ok thanks
I have to remove all the line ? is it correct ?